sdk/typescript/src/cryptography/ed25519-keypair.ts

```diff
@@ -100,4 +109,20 @@ export class Ed25519Keypair implements Keypair {
       nacl.sign.detached(data.getData(), this.keypair.secretKey)
     );
   }
-}
\ No newline at end of file
+
+  /**
+   * Derive Ed25519 keypair from mnemonics and path.
+   * The path must be compliant to SLIP-0010 in form m/44'/784'/{account_index}'/{change_index}'/{address_index}'
+   */
+  static deriveKeypair(path: string, mnemonics: string): Ed25519Keypair {
```

Comment:
IMO we should flip the argument order and make the path option, defaulting into the default path.